src/test/lrsql/admin/route_test.clj

```diff
@@ -272,6 +272,49 @@
             (is (= 400 status))))))
     (component/stop sys')))
 
+(def proxy-jwt-body
+  {"usercertificate" "unique.user.1234"
+   "iss"             "https://idp.domain.com/auth"
+   "group-full"      ["/domain/app/ADMIN"]})
+
+(defn- proxy-jwt
+  "Takes an edn claims body and returns a (not properly signed!) JWT
+   for proxy jwt testing"
+  [body]
+  (str "eyJhbGciOiJIUzI1NiJ9."
+       (u/str->base64encoded-str (u/write-json-str body))
+       ".GLkxNsaxkZQiW4voy4RKEpLy8RxyzcpMBAeBw-aIykQ"))
+
+(deftest proxy-jwt-admin-routes
+  (let [sys  (support/test-system
+              :conf-overrides
+              {[:webserver :jwt-no-val]          true
+               [:webserver :jwt-no-val-uname]    "usercertificate"
+               [:webserver :jwt-no-val-issuer]   "iss"
+               [:webserver :jwt-no-val-role-key] "group-full"
+               [:webserver :jwt-no-val-role]     "/domain/app/ADMIN"})
+        sys' (component/start sys)
+        ;; proxy jwt auth
+        proxy-auth {"Authorization" (str "Bearer " (proxy-jwt proxy-jwt-body))}
+        headers  (merge content-type proxy-auth)]
+    (testing "Proxy JWT authentication"
+      ;; Run a basic admin routes call and verify success
+      (let [{:keys [status body]} (get-account headers)
+            edn-body (u/parse-json body :object? false)]
+        ;; 200 response
+        (is (= status 200))
+        ;; not only is there a body but it should now contain our jwt user
+        (is (some #(= (get % "username") (get proxy-jwt-body "usercertificate"))
+                  edn-body))))
```

Comment:
In addition to this `get-account` test, we should also have tests that uses this JWT to log in and do admin actions like account updating (unless I'm misunderstanding how these special JWTs work).

Reply:
it is already logging in and doing admin actions. There isnt a "login" per se, and get-account is admin get-all-accounts. And there are no difference in roles in lrs admin ops. If you can do one thing you can do everything.

Reply:
Ah okay. I'd personally add more tests just to feel safe, but I see what you mean by `get-account` being enough for admin testing.

That said, since you can't create these JWTs by conventional login (making it more akin to setting the initial seed admin username/password in the config), can you please remind how they are intended to be created in the first place?

Reply:
I can't on this channel, answering in slack.